toolchain/driver/llvm_subcommand.h

```diff
@@ -18,8 +18,11 @@ namespace Carbon {
 //
 // See the implementation of `Build` for documentation on members.
 struct LLVMOptions {
-  // Build the LLVM subcommand options using `b`. Whatever eventual nested
-  // subcommand action is selected, it needs to additionally run `select_fn`.
+  // Build the LLVM subcommand options using `b`.
+  //
+  // When this top-level subcommand is selected (potentially through a nested
+  // sub-subcommand), the `selected_subcommand` should be set to point to
```

Comment:
```suggestion
  // sub-subcommand), the `selected_subcommand` will be set to point to
```

(In my experience, "should" typically imposes a requirement on the caller, whereas requirements on the function are just stated as facts.)